help/destinations/catalog/advertising/liveramp-distribution.md

```diff
@@ -158,39 +189,117 @@ To configure details for the destination, fill in the fields below.
+>id="platform_destinations_liveramp_distribution_nexxen_advertiserid"
+>title="Advertiser ID"
+>abstract="If you're sending data to a single advertiser in the Amobee platform, enter the Amobee Advertiser ID. It should be completely numeric. If you want to make the data available to all advertisers in a market or if these segments are "AlwaysOn", please enter -1."
+
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_nexxen_contactemail"
+>title="Contact Email"
+>abstract="Enter the best email(s) for Amobee to send data contract details. Most likely this is your own email address, but it can also be an email alias. For multiple recipients, separate using commas (i.e. email1@domain.com,email2@domain.com,etc)."
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Nexxen destination.](../../assets/catalog/advertising/liveramp-distribution/LR_Nexxen_DestSpecific.png)
+
+* **[!UICONTROL Rate Type]**: Rate type represents the way data usage should be billed. All $0.00 rates should be flat fee. Please confirm with your Amobee representative if you're not sure which rate type to use.
+* **[!UICONTROL Market ID]**: Enter the Market ID where the Amobee data contract should be created. It should be completely numeric. If you are doing "AlwaysOn" syndication across every single market in the Amobee platform, enter -1.
+* **[!UICONTROL Advertiser ID]**: If you're sending data to a single advertiser in the Amobee platform, enter the Amobee Advertiser ID. It should be completely numeric. If you want to make the data available to all advertisers in a market or if these segments are "AlwaysOn", please enter -1.
+* **[!UICONTROL Contact Email]**: Enter the best email(s) for Amobee to send data contract details. Most likely this is your own email address, but it can also be an email alias. For multiple recipients, separate using commas (i.e. email1@domain.com,email2@domain.com,etc).
+
+### [!DNL One Fox] {#fox}
+
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_fox_client"
+>title="Client"
+>abstract="The name of your company/distribution account as you would like it to appear to the partner. Use your company name by default, but please contact your partner account rep if unsure. No whitespace or other special characters, please (e.g. Company_Name)."
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the One Fox destination.](../../assets/catalog/advertising/liveramp-distribution/LR_Fox_DestSpecific.png)
+
+* **[!UICONTROL Client]**: The name of your company/distribution account as you would like it to appear to the partner. Use your company name by default, but please contact your partner account rep if unsure. No whitespace or other special characters, please (e.g. Company_Name).
```

Comment:
* **[!UICONTROL Client]**: The name of your company/distribution account as you would like it to appear to the partner. Use your company name by default. If unsure, please contact your partner account representative. Do not use spaces or special characters (e.g. Company_Name).